src/hotspot/share/cds/archiveUtils.inline.hpp

```diff
@@ -25,7 +25,7 @@
 #ifndef SHARE_MEMORY_ARCHIVEUTILS_INLINE_HPP
```

Comment:
The header file include guards all need updating for the new path.

Reply:
Fixed.